### DISCUSSION TITLE
Stream Deck FiveM Integration?

### DISCUSSION THREAD

**Discussion by AverageGamerStudios** [elgatosf/elgato#97] [General]
FiveM is a moddified version of Grand Theft Auto V, making it so you can create your own servers, be it custom like GTA Online, or for Roleplay. Question is, is it possible to create Stream Deck Integrated into FiveM? I am wanting to create my own server, and just wondering if this is possible. I want to have keys for emotes (animations), commands (without interrupting the pace of gameplay) and other mechanics. Please let me know!

**Comment by Firecul** [comment]
You can get the stream deck to trigger autohotkey scripts that can send your commands already, that's what I used to do.

**Comment by iWanheda** [comment]
https://docs.fivem.net/natives/?_0xD7664FD1
